### PR TITLE
Create Log In Method, Login Form and Implement Front-end Authentication

### LINKED ISSUE #2: [1pt] Create Log In Method and Form
- Create a corresponding form in the React front-end for username and password inputs.

### LINKED ISSUE #3: [1pt] Implement Front-end Authentication
- Develop the functionality on the React front-end to authenticate users using the 
provided username and password.

### PR DESCRIPTION
**Pull request by SabaAhmad404**
In this branch , we(@SabaAhmad404 and @Lucash2022 ) worked on

- Create a corresponding form in the React front-end for username and password inputs.

- Develop the functionality on the React front-end to authenticate users using the 
provided username and password.
image

### CODE REVIEW

**Review by FaranoshAmini — APPROVED** [review]

**Review by FaranoshAmini — APPROVED** [review]
## Hi Team 👋🏻 
# STATUS : APPROVED 

Your project is complete! There is nothing else to say other than... it's time to merge it 

### Congratulations! 🎉
## **Highlights**
✔Correct workflow was used 
✔️Well organized and clean project 
✔️Good project structure
 ✔️Good implementation of sign in
✔️Detailed pull request message
 ✔️Good commit history 

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @FaranoshAmini  in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

File: src/redux/store.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import { configureStore } from '@reduxjs/toolkit';
 import MoviesSlice from './MoviesSlice';
+// eslint-disable-next-line import/extensions
+import userReducer from './UsersSlice';
```
Comment (torobucci) [inline src/redux/store.js:3]: Hello @SabaAhmad404  and @Lucash2022 
Kindly ensure the `userReducer` is imported from the correct file path in order for the app to work as intended

Comment (Lucas-Erkana) [reply src/redux/store.js:3]: Hello @torobucci,

Thank you so much for taking the time to review our code and providing valuable feedback. Your keen eye helped us identify the issue with the userReducer import, and we truly appreciate your input.

We have carefully looked into the problem you pointed out and have successfully resolved it by renaming the file from Userslice.js to UsersSlice.js. This change has been committed, and you can find the details in this commit: [Commit Link](https://github.com/Lucash2022/BookFlix-Frontend/pull/16/commits/146f4890ad5a091fc606b451f94e821cf605f2d5).

Thanks to your review, the app is now working as intended, and the userReducer is imported correctly.

Once again, we are grateful for your help in improving our project. Your feedback has been invaluable to us. If you have any more suggestions or feedback in the future, please don't hesitate to share it with us. We look forward to working together to enhance our codebase.

Best regards,
@Lucash2022 and @SabaAhmad404

**Review by SamTush — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @Lucash2022 & @SabaAhmad404, 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  a. 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x] Login in functioning as required 😎 
- [x] No linter errors
- [x] App works and runs as intended 💯
 

 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------

**Pull request merged**